src/_utils/focusVisible/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { PropsWithChildren, createContext, useEffect, useState } from 'react'
```

Comment:
Can you name the folder focusVisibleProvider so we know that it's a provider and to keep it consistent with mediaSizeProvider

Reply:
Yep done